models/Size.js

```diff
@@ -6,8 +6,12 @@ const SizeSchema = new mongoose.Schema({
   widthInCm: Number,
```

Comment:
I think we can store it `width: String`. The actual value will look like "12cm" because I don't think that we are going to perform some calculations using width and number will be useless here

Reply:
That would require applying the "language" structure (12cm / 12см). Also, is it "12cm" or "12 cm"? What if someone down the line wants to convert it after all? What are the benefits of going against 1NF like that?